proto/spacemesh/global_state.proto

```diff
@@ -10,45 +11,45 @@ service GlobalStateService {
     // Latest computed global state - layer and its root hash
     rpc GlobalState(google.protobuf.Empty) returns (GlobalState);
 
-    // Get the current transaction state. Note that tx state is not part of
-    // global state. It is metadata about a transaction on its journey to the
-    // mesh and from the mesh to the global state (or to null)
+    // Returns current transaction state which describes its journey
+    // to the mesh and from the mesh to the global state (if it got processed)
     rpc TransactionState(TransactionId) returns (TransactionState);
 
-    // Account info in the current global state
+    // Returns the current state of one or more transactions
+    rpc TransactionsState(TransactionsIds) returns (TransactionsState);
```

Comment:
What's the advantage of having this, over just allowing repeated calls to `TransactionState`?

Reply:
avoiding having to issue multiple calls to TransactionState() when client has more than one tx id it wants results for. And the other version when state of only 1 transaction is expected. So users won't have to use TransactionsState() in this case... it is not huge deal - I think it is nice and anticipates the calling patterns.

Reply:
I see your point but I suppose I prefer keeping the API as simple as possible, and also symmetry across different data types - e.g, why allow a query for multiple transactions but not for multiple accounts?

If we have `TransactionsState` then why do we need `TransactionState` at the same time? Can't you just call `TransactionsState` with a single txid?

Reply:
ok, I removed the help connivance method.